packages/fetchai/connections/p2p_libp2p/dht/dhtpeer/dhtpeer.go

```diff
@@ -671,12 +675,28 @@ func (dhtPeer *DHTPeer) handleAeaAddressStream(stream network.Stream) {
 }
 
 func (dhtPeer *DHTPeer) handleAeaNotifStream(stream network.Stream) {
-	lerror, _, linfo, _ := dhtPeer.getLoggers()
+	lerror, _, linfo, ldebug := dhtPeer.getLoggers()
 
 	linfo().Str("op", "notif").
 		Msgf("Got a new notif stream")
 
 	if !dhtPeer.addressAnnounced {
+		// workaround: to avoid getting `failed to find any peer in table`
```

Comment:
Fixes race condition that causes libp2p `test_communication.py` to be flaky.
It seems that the race condition is well know to libp2p [1].
They even need to consider it in their own tests [2]

[1] https://github.com/libp2p/go-libp2p-kad-dht/issues/308
[2] https://github.com/libp2p/go-libp2p-kad-dht/blob/v0.7.11/dht_test.go#L179